Playground/kotlin-starter-app/app/src/main/java/com/runanywhere/kotlin_starter_example/ui/screens/SpeechToTextScreen.kt

```
@@ -0,0 +1,461 @@
+                while (isRecording) {
+                    val read = audioRecord?.read(buffer, 0, buffer.size) ?: 0
+                    if (read > 0) {
+                        synchronized(audioData) {
+                            audioData.write(buffer, 0, read)
+                        }
+                    }
+                }
+            }.start()
+            
+            return true
+        } catch (e: SecurityException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)